### PR TITLE
Add C4E chain and uc4e asset.

### PR DESCRIPTION
**Pull request by pablokulpa**
### Adding Chains

If adding a new chain, please ensure the following:
- [X] Chain is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - Chain's registration must have `staking` defined, with at least one `staking_token` denom specified --> https://github.com/cosmos/chain-registry/pull/4015
   - Chain's registration must have `fees` defined; at least one fee token has low, average, and high gas prices defined. --> https://github.com/cosmos/chain-registry/pull/4015
- [X] Add chain to bottom of `zone_chains.json`
   - RPC and REST must not have any CORS blocking of the Osmosis domain.
   - RCP node must have WSS enabled.

### Adding Assets

If adding a new asset, please ensure the following:
- [X] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [X] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry PR: https://github.com/cosmos/chain-registry/pull/4014
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: ______
- [ ] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [X] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis.zone_chains.json [hunk 1]
```diff
@@ -1141,6 +1141,16 @@
         "eth-key-sign"
       ]
     },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_chains.json:1144]: Please move to bottom of the list

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -1236,6 +1236,14 @@
       "peg_mechanism": "collateralized",
       "osmosis_verified": true
     },
+    {
```
[the remark this answers is not in the document]
Comment (pablokulpa) [reply osmosis-1/osmosis.zone_assets.json]: ok done.

File: osmosis-1/osmosis.zone_chains.json (hunk 1 above)
Comment (pablokulpa) [reply osmosis-1/osmosis.zone_chains.json]: ok done.

**Comment by JeremyParish69** [comment]
When is the planned stream?

**Pull request merged**